### PR TITLE
Project Clean up...

### PR DESCRIPTION
**Pull request by daceves**
* Removed print_r()
* Removed exit(). Avoid using exit, throwing exception instead.
* Changing logical operators from 'and' to '&&' to avoid potential unintended behavior
* Adding 'use' for Symfony\Component\Yaml\Exception\ParseException; To avoid confusion with Symfony\Component\CssSelector\Exception\ParseException
* Removed executable permission on config/translation/vn/*.yml files

### CODE REVIEW

**Comment by jmolivas** [comment]
@daceves: thanks for the PR, I will be looking at this today

**Comment by darol100** [comment]
:+1:  For this PR.  I do not see anything wrong with it.

**Comment by jmolivas** [comment]
Actually is a pretty nice PR ;)

**Comment by daceves** [comment]
Thanks. :+1:

File: src/Command/Self/UpdateCommand.php [hunk 1]
```diff
@@ -124,6 +124,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
         // can't read files properly from the overwritten Phar, or perhaps it's
         // because the autoloader's name has changed. We avoid the problem by
         // terminating now.
-        exit;
+        throw new Exception('Unexpected instantiation');
```
Comment (jmolivas) [inline src/Command/Self/UpdateCommand.php]: If we do not have an exit here the self-update command output an exception after updating the project.

**Comment by jmolivas** [comment]
I sent another PR https://github.com/hechoendrupal/DrupalConsole/pull/2033 including your changes + other improvements.

**Pull request merged**